### PR TITLE
fix(event severity): remove event severity subtype

### PR DESCRIPTION
**Pull request by juliayakovlev**
According to new SCT Events implementation, severity subtype
should be removed.
It impacts on ClusterHealthValidatorEvent and DataValidatorEvent

Existing unit tests were changed and passed

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/sct_events/health.py [hunk 1]
```diff
@@ -23,11 +23,13 @@ class ClusterHealthValidatorEventClusterHealthCheck(SctEventProtocol, Protocol):
...
-    NodesNemesis: Type[SeverityLevelProtocol]
+class ClusterHealthValidatorEvent(SctEvent, abstract=True):
+    NodeStatus: Type[SctEventProtocol]
+    NodePeersNulls: Type[SctEventProtocol]
+    NodeSchemaVersion: Type[SctEventProtocol]
+    NodesNemesis: Type[SctEventProtocol]
+    Done: Type[SctEventProtocol]
+    Info: Type[SctEventProtocol]
 
     ClusterHealthCheck: Type[ClusterHealthValidatorEventClusterHealthCheck]
```
Comment (enaydanov) [inline sdcm/sct_events/health.py:34]: I think you should need to remove this subevent (and Protocol used here.)

File: sdcm/sct_events/health.py [hunk 2]
```diff
@@ -45,27 +47,27 @@ def __init__(self,
...
-ClusterHealthValidatorEvent.add_subevent_type_with_severity_levels("NodeSchemaVersion")
-ClusterHealthValidatorEvent.add_subevent_type_with_severity_levels("NodesNemesis")
+ClusterHealthValidatorEvent.add_subevent_type("NodeStatus")
+ClusterHealthValidatorEvent.add_subevent_type("NodePeersNulls")
+ClusterHealthValidatorEvent.add_subevent_type("NodeSchemaVersion")
+ClusterHealthValidatorEvent.add_subevent_type("NodesNemesis")
+ClusterHealthValidatorEvent.add_subevent_type("Done", severity=Severity.NORMAL)
+ClusterHealthValidatorEvent.add_subevent_type("Info", severity=Severity.NORMAL)
 
 ClusterHealthValidatorEvent.add_subevent_type("ClusterHealthCheck", abstract=True)
```
Comment (enaydanov) [inline sdcm/sct_events/health.py:63]: Please, remove it.

**Review by enaydanov — CHANGES_REQUESTED** [review]
Please, fix unit tests

File: sdcm/sct_events/health.py (hunk 1 above)
Comment (juliayakovlev) [reply sdcm/sct_events/health.py]: removed

File: sdcm/sct_events/health.py (hunk 2 above)
Comment (juliayakovlev) [reply sdcm/sct_events/health.py]: removed

File: unit_tests/test_sct_events_health.py [hunk 3]
```diff
@@ -27,35 +28,34 @@ def test_cluster_health_validator_event(self):
         self.assertTrue(issubclass(ClusterHealthValidatorEvent.NodeSchemaVersion, ClusterHealthValidatorEvent))
         self.assertTrue(hasattr(ClusterHealthValidatorEvent, "NodesNemesis"))
         self.assertTrue(issubclass(ClusterHealthValidatorEvent.NodesNemesis, ClusterHealthValidatorEvent))
-        self.assertTrue(hasattr(ClusterHealthValidatorEvent, "ClusterHealthCheck"))
-        self.assertTrue(issubclass(ClusterHealthValidatorEvent.ClusterHealthCheck, ClusterHealthValidatorEvent))
```
Comment (enaydanov) [inline unit_tests/test_sct_events_health.py:31]: Please, add checks for `Info` and `Done` subevents.

**Review by enaydanov — CHANGES_REQUESTED** [review]

File: unit_tests/test_sct_events_health.py (hunk 3 above)
Comment (juliayakovlev) [reply unit_tests/test_sct_events_health.py:31]: done

**Review by enaydanov — APPROVED** [review]

**Pull request merged**